src/components/Input.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FC } from "react";
+
+interface IProps {
+  name: string;
+}
+
+export const Input: FC<IProps> = ({ name }) => (
+  <div>
+    <p className="text-sm text-grey font-lato mb-2 mt-3">{name}</p>
+    <input
+      type="text"
+      placeholder="Zadajte Váš email"
+      className={`w-full h-10 font-lato text-base text-sm hover:shadow-lg pl-4`}
```

Comment:
no need for the brace string literal syntax, just simply ""

Reply:
done